### ISSUE TITLE
Is it possible to unlink a HS Replay account ?

### ISSUE THREAD

**Issue by Gourmont** [HearthSim/Hearthstone-Deck-Tracker#3348]
Hello,
I apologize if this question has already been asked : I linked a wrong HSReplay account to HDT and I can't unlink it now... I tried uninstalling HDT and installing it again but it still remembers which HSReplay I linked it to. Is there any way to remove it and link it to another HSReplay account ?

Thanks in advance !

**Comment by NightKev** [comment]
Yes, I believe so. Go to https://hsreplay.net/account/social/connections/ when you're logged in to the site and you can change which Blizzard accounts you have connected to HSReplay (and which one you want set as the primary account). The page should look something like this: 
![hsreplay_blizz_account_settings](https://user-images.githubusercontent.com/849285/28810555-3587efbe-763f-11e7-81a9-d6e727f3008c.png)

**Comment by Gourmont** [comment]
Thanks for your answer but the problem is not about HSReplay itself, I am going to explain a bit deeper : 
My friend and I shared the same HSReplay account together. Now, we want to play on separate accounts, so I "removed" her name from my account (I had to go to the page you linked) and she created her own HSReplay account with her battle tag. But, in her HDT, she is still linked to my HSReplay account (the image below) and every game she plays keeps being updated to my HSR account instead of her.
Tl; dr : She would like to know how to have her name instead of mine on the screenshot below.

https://puu.sh/wYtjX/b938ad6f44.png

**Comment by Gourmont** [comment]
Problem solved ! After uninstalling HDT, we had to go to data>local>roaming and delete the HDT file to be able to claim again the HSR account ! Thanks !

**Comment by jleclanche** [comment]
Yeha you just have to delete one file actually (don't remember which one). `hsreplay.cache` or some such?

@Gourmont If there's replays to reassign please make sure to email support@hearthsim.net with account details, by the way!
